custom_nav2:
  ros__parameters:

    # Initial pose [x, y, yaw] in "map" frame
    initial_pose: [0.0, 0.0, 0.0]

    # Final target pose [x, y, yaw] in "map" frame
    target_pose: [8.0, -0.5, 1.57]

    # Default intermediate waypoint [x, y, yaw]
    # (will be overridden if /traffic_waypoint is received)
    waypoint: [4.0, 0.8, 0.0]

    # Time (seconds) to wait for /traffic_waypoint
    wait_for_traffic_wp: 5.0
